README.md

```diff
@@ -14,36 +14,36 @@
 
 - Grant execute permission to install script.
 
-  > `sudo chmod 777 install.sh`
+  > `chmod a+x install.sh`
 
 - Run install script to configure client.
 
-  > `sudo ./install.sh`
+  > `./install.sh`
 
 ### **Server Setup**
 
 - Switch to server folder.
 
   > `cd server`
 
-- Start server script with pm2.
+- Start server script.
 
-  > `pm2 start main.js -n "name your server instance"`
-
-- To view server log run pm2 log "number assigned to server instance by pm2 here".
-
-  > `pm2 log 0`
+  > `start main.js`
 
 ### **Client Setup**
 
 - Switch to client folder.
 
   > `cd ../client`
 
-- Start client script with pm2.
+- Start client script.
+
+  > `start main.js`
+
```

Comment:
No need for this aspects now....
npm run start-server
npm run start-client
For starting or running the clients respectively.

Check the pm2 alternatives to starting npm scripts

Reply:
Npm scripts. This works "start-server" : "cd server && node main.js" and "start-client":"cd client && node main.js". added to package.json